xbmc/peripherals/devices/PeripheralCecAdapter.cpp

```diff
@@ -1385,6 +1385,7 @@ void CPeripheralCecAdapter::SetConfigurationFromSettings(void)
   m_bPowerOffScreensaver               = GetSettingBool("cec_standby_screensaver") ? 1 : 0;
   m_bPowerOnScreensaver                = GetSettingBool("cec_wake_screensaver") ? 1 : 0;
   m_bSendInactiveSource                = GetSettingBool("send_inactive_source") ? 1 : 0;
+  m_configuration.bAutoWakeAVR         = GetSettingBool("power_avr_on_as") ? 1 : 0;
```

Comment:
It is. This is read from Kodi's config only. I got rid of all the unneeded stuff in `SetConfigurationFromLibCEC()`